client/me/purchases/downgrade/style.scss

```diff
@@ -77,6 +86,22 @@
 
 	.form-button {
 		margin-right: 8px;
+
+		&[disabled] {
+			animation: loading-button-animation 1.6s infinite linear;
+			background-color: var( --color-neutral-10 );
+			background-image: linear-gradient(
+				-45deg,
+				var( --color-neutral-10 ) 28%,
+				var( --color-neutral-0 ) 28%,
+				var( --color-neutral-0 ) 72%,
+				var( --color-neutral-10 ) 72%
+			);
+			background-size: 50% 100%;
+			background-repeat: repeat-x;
+			border-color: var( --color-neutral-5 );
+			pointer-events: none;
+		}
```

Comment:
Generally we should try to revert to default styles. I think `@wordpress/components` has a Button component with an `isBusy` flag we can use :)

Reply:
Good call. Thanks!